README.md

```diff
@@ -76,6 +76,22 @@ The project is structured with the following in mind:
 - tests/unit/
     - pytest tests
 
+---
+## Deployment
+
+We use a variety of technologies to get this code into production.  Starting from this repo and going outwards:
+
+1. github actions builds and deploys a docker container to ecr
+    1. prs and pushes to this repo will build and push a 'short-sha' container to AWS' ECR
+    1. Code merged to main will trigger a build that prefixes the short sha with literal 'stg-'
+    1. Code merged to prod will trigger a build that prefixes the short sha with literal 'prod-'
```

Comment:
Can we use git tags to identify production code instead of a branch ``prod``?

If we need to use a branch, then we may need some GitHub settings to protect that branch.